### DISCUSSION TITLE
제 커뮤니티 모각코 안내

### DISCUSSION THREAD

**Discussion by jongfeel** [humansof42seoul/Humansof42-v2#7] [Show and tell]
@jwon42 
@42hm 
@joho3116 

제가 github id를 여기까지 찾긴 했는데 jaewpark님의 github id를 못찾겠네요.

어제 피자먹으면서 얘기했던 제 커뮤니티의 온라인 모각코는 아래 입니다.

https://github.com/ThinkAboutSoftware/OnlineSelfCodingGroup

그리고 제 커뮤니티에 관심 있으신 분 역시 같이 참여해주시면 좋을 것 같아요!

**Comment by jwon42** [comment]
감사합니다.
jaewpark님 깃허브 아이디는 @ohdair 입니다.
